common/startup/startup.go

```diff
@@ -126,12 +128,17 @@ func GetLogic(backend *ecsbackend.ECSBackend) (*logic.Logic, error) {
 
 func getNewTagStore() (tag_store.TagStore, error) {
 	creds := credentials.NewStaticCredentials(config.AWSAccessKey(), config.AWSSecretKey(), "")
-	awsConfig := &aws.Config{
-		Credentials: creds,
-		Region:      aws.String(config.AWSRegion()),
+	session := session.New(config.GetAWSConfig(creds, config.AWSRegion()))
```

Comment:
shouldn't this use the new helper?

Reply:
The helper is `config.GetAWSConfig()`, are you suggesting that I should make the helper get the entire Session object as opposed to getting the AWS configuration?

Reply:
I'm suggesting that this block of logic should not be copy+pasted; it should only exist in a single location of our codebase. How you accomplish that is up to you.

Reply:
:white_check_mark: I implemented a helper function inside this file to handle the redundant logic.